Seccomp Update the existing one and add weaker config

surajssd opened the pull request.

alban commented on seccomp.go:
```diff
@@ -1,1367 +1,1602 @@
-					"value": 0,
-					"valueTwo": 0,
-					"op": "SCMP_CMP_EQ"
+	seccompDefaultProfile = `{
```
Could you add a comment with a link saying where this was taken from?

And for seccompWeakProfile, a comment saying what's the difference?

Maybe those two constants should be in different files (seccomp-default.go and seccomp-weak.go), that would make it easier to run `diff` between the two.

surajssd replied:
done

surajssd committed:
seccomp: Update default profile

The latest profile downloaded at:
https://github.com/moby/moby/blob/238f8eaa31aa74be843c81703fabf774863ec30c/profiles/seccomp/default.json

Signed-off-by: Suraj Deshmukh <suraj@kinvolk.io>

surajssd committed:
seccomp: Add weaker profile variable

This commit adds a weaker seccomp profile as a different variable which
allows `unshare` syscall without any requirement of any capability.

```
{
    "names": [
        "unshare"
    ],
    "action": "SCMP_ACT_ALLOW",
    "args": [],
    "comment": "",
    "includes": {},
    "excludes": {}
},
```

Signed-off-by: Suraj Deshmukh <suraj@kinvolk.io>

surajssd commented:
I am adding weaker seccomp profile but it will be consumed only when the code will be added. Right now going forward with merging it?

indradhanush commented:
@surajssd Yes, let's get this merged in favor of velocity. We can come back to iterate and fix issues.
